docker/Dockerfile-opflex

```diff
@@ -1,7 +1,7 @@
 ARG basetag=latest
 ARG baserepo=quay.io/noirolabs
-FROM ${baserepo}/aci-containers-base:${basetag}
-RUN yum install -y --enablerepo=openstack-15-for-rhel-8-x86_64-rpms \
+FROM ${baserepo}/aci-containers-base:kmr2-test
```

Comment:
We should not be changing this, it will automatically get set to the relevant base image from the Jenkins jobs.

Reply:
yes, I will revert this back later but for current testing purpose to build an image via jenkins job using this branch had to tweak this. 

Image -> https://noiro-quay.cisco.com/repository/noiro/opflex?tab=tags health-index-test

job used with names -> aci-containers-opflex and  aci-containers-opflex-build-base in  https://engci-jenkins-sjc.cisco.com/jenkins/job/team_noiro_engineering/view/Users/job/Vishal/ 

@snaiksat